openquake/risklib/scientific.py

```diff
@@ -1129,7 +1129,9 @@ def insured_losses(losses, deductible, insurance_limit):
         deductible = numpy.full_like(losses, deductible)
     if not isinstance(insurance_limit, numpy.ndarray):
         insurance_limit = numpy.full_like(losses, insurance_limit)
-    assert (deductible < insurance_limit).all()
+    assert (deductible < insurance_limit).all(), (
+        "Please check deductible values. Values larger than the"
+        " insurance limit were found.")
```

Comment:
It would be nice to print out the asset IDs for which the values are too large.

Reply:
Ok, I will fix that

Reply:
It is less trivial than expected, because it would require to test `insurance_losses`in `openquake/risklib/scientific.py` (currently only `insured_losses` is tested).

Reply:
It is tested:
```bash
 $ OQ_DISTRIBUTE=no pytest openquake/calculators/tests/event_based_risk_test.py  -k 1_ins
```
put a pdb inside and you will see the asset IDs:

```(Pdb++) asset_df
            id        lon        lat  ...  ideductible  taxonomy  policy
ordinal                               ...                               
0        b'a0'  81.298500  29.109800  ...          0.0         1       2
3        b'a3'  85.747704  27.901501  ...          0.0         1       1
```

Reply:
Thank you @micheles

Reply:
Done